core/win32/syscall.c

```diff
@@ -1710,7 +1710,11 @@ presys_TerminateProcess(dcontext_t *dcontext, reg_t *param_base)
                                    THREAD_SYNCH_VALID_MCONTEXT_NO_XFER,
                                    /* if we fail to suspend a thread (e.g., privilege
                                     * problems) ignore it. FIXME: retry instead? */
+                                   /* FIXME i#2345: add THREAD_SYNCH_SKIP_CLIENT_THREAD
+                                    * to synch and kill application threads only.
```

Comment:
The comment is a little confusing b/c this is not currently killing *any* threads